csrankings-k.csv

```diff
@@ -338,7 +338,7 @@ Kay Nieselt,University of Tübingen,https://uni-tuebingen.de/en/faculties/facult
 Kay Nieselt-Struwe,University of Tübingen,https://uni-tuebingen.de/en/faculties/faculty-of-science/departments/computer-science/lehrstuehle/integrative-transcriptomics/group,Ty-5Q7sAAAAJ
 Kay Robbins,University of Texas at San Antonio,http://www.cs.utsa.edu/~krobbins,g0dz7JoAAAAJ
 Kayhan M. Imre,Hacettepe University,https://web.cs.hacettepe.edu.tr/~kimre,1d2qMREAAAAJ
-Kayhan N. Batmanghelich,University of Pittsburgh,https://kayhan.dbmi.pitt.edu,PvHFAfIAAAAJ
+Kayhan N. Batmanghelich,Boston University,https://www.bu.edu/eng/profile/kayhan-batmanghelich-ph-d/,PvHFAfIAAAAJ

```

Comment:
Please provide evidence of a CS affiliation or other means to meet the inclusion criteria.